vignettes/get_data_meetnetten.Rmd

```diff
@@ -0,0 +1,296 @@
+
+locations_dragonflies$main_locations %>%
+  ggplot() +
+  geom_sf(aes(fill = scheme, colour = scheme), alpha = 0.5) 
+```
+
+## Visits
+
+The function `get_meetnetten_visits` returns all visits (count events) for 
+one or more monitoring schemes.
+
+Below we select all visits for the location Arenbergpolder of the Argusvlinder
+monitoring scheme. 
+We see that in most years the location is counted 6 times per year as demanded
+by the [monitoring protocol](https://purews.inbo.be/ws/portalfiles/portal/17657883/MaesEtal_2019_MonitoringsprotocolDagvlindersVersie2.pdf).
+
+```{r example_visits}
+
+visits <- get_meetnetten_visits(con,
+                                scheme_name = "Argusvlinder",
+                                collect = TRUE)
+
+```
+
+```{r, eval = TRUE}
+visits %>%
+  filter(location == "Arenbergpolder") %>%
+  select(location, visit_id, start_date, 
+         start_time, visit_status)
+```
+
+Use `?get_meetnetten_visits` to get more information on the variables that are
+returned by the function.
+
+## Observations
+
+The function `get_meetnetten_observations` provides all observations for one 
+or more monitoring scheme.
+
+Let's select all observations from one visit at the Arenbergpolder location
+of the Argusvlinder monitoring scheme, by applying a filter based on the
+variable `visit_id`.
+
+```{r example_observations}
+
+observations <- get_meetnetten_observations(con,
+                                            scheme_name = "Argusvlinder",
+                                            collect = FALSE) %>%
+  filter(visit_id == 1080) %>%
+  collect()
+
+```
+
+First we take a look at the observation of the target species.
+
+```{r, eval = TRUE}
+
+observations %>%
+  filter(target_species) %>%
+  select(location, sublocation, start_date, 
+         scientific_name, count)
+```
+Optionally observers can also record other species that can be counted using the
+same protocol.
+
+The code below provides the total number of individuals per species,
+including the non target species, that were counted in one visit at the
+Arenberpolder location.
+
+```{r, eval = TRUE}
+
+observations %>%
+  group_by(start_date, target_species, name_nl, scientific_name) %>%
+  summarise(count_total = sum(count)) %>%
+  ungroup()
+```
+
+Use `?get_meetnetten_observations` to get more information on the variables that
+are returned by the function. 
+
+An important note: before you start analysing the data from a certain monitoring
+scheme, it is important to know how the monitoring schemes were designed and how
+the data is organised.
+So don't forget to check the monitoring protocol (available on the 
+[INBO website](https://www.vlaanderen.be/inbo/zoeken/?q=monitoringsprotocol)).
+
+## Closing the connection
+
+Close the connection when done
+
+```{r closing_conn}
+dbDisconnect(con)
+rm(con)
+```
+
+```{r echo=FALSE, include=FALSE}
+rm(database_access)
+save.image(file = "get_data_meetnetten_results.Rda")
+```
+
+# Data policy
```

Comment:
Wat met projecten met beperkte toegang, zoals de wintertellingen vleermuizen? Moeten we niet meegeven dat die vleermuizendata zeker nooit zomaar gepubliceerd mogen worden zonder overleg met de vleermuiswerkgroep?

Reply:
Suggestie: als je zeker wil zijn dat iedereen die boodschap krijgt, overweeg dan om (ook) een warning te geven ingeval die gegevens opgevraagd worden.  Ga er zeker niet van uit dat iedereen het vignet leest en de volledige inhoud onthoudt tot het eens van toepassing is. ;-)

Reply:
In de tutorial houd ik het graag zo generiek mogelijk. Ik heb nu toegevoegd om altijd info@meetnetten.be te contacteren voor resultaten te publiceren. Zo kunnen we nagaan of de publicatie conform is met de afspraken die dan geldig zijn. Meer details kunnen we dan op het infomoment geven.

Reply:
Lees jij altijd een tutorial vooraleer je een package gebruikt?  Eerlijk gezegd: als je het echt belangrijk vindt dat gebruikers die boodschap krijgen, dan zou ik die boodschap toch ergens geven bij het runnen van de functie(s).  Dit moet geen warning zijn, het mag gewoon een 'message' zijn die verschijnt tijdens het runnen van een `get_meetnetten`-functie.  (Doordat het maar over een deel van de functies van inbodb gaat, lijkt een opstartboodschap bij inbodb me minder geschikt, anders was dat ideaal geweest.)  Tidyverse heeft soms boodschappen die eenmaal om de 8 uur verschijnen, misschien is zoiets een ideetje om de gebruiker niet te vervelen met constante boodschappen en ze er toch bij elk gebruik even aan te herinneren?